crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -693,6 +694,16 @@ async fn forward_software_request(
     )])
 }
 
+fn get_smartrest_template(payload: &str) -> Result<String, SmartRestDeserializerError> {
+    Ok(payload
+        .split(',')
+        .next()
+        .ok_or_else(|| SmartRestDeserializerError::InvalidMessagePayload {
+            payload: payload.to_string(),
+        })?
```

Comment:
This could safely be replaced by an `unwrap` as the first element of the split will be the whole payload if there is no comma.

Reply:
Yes, make sense